### ISSUE TITLE
Create Collaboration Cycle Zenhub board

### ISSUE THREAD

**Issue by chaseakirby** [department-of-veterans-affairs/va.gov-team#34738]
## User Story
As a Governance Team member, I want to see Collaboration Cycle touchpoints on a Kanban Board so that I can anticipate upcoming reviews.

## Description
A Collaboration Cycle Kanban board existed previously but was not found to be useful. 
This new board is intended to utilize the Core GitHub ticket to allow for statusing while consolidating the touchpoint tickets into one for a team/product's journey through the Collaboration Cycle for a specific release or iteration.

## Artifacts Needing Updates
[Collaboration Cycle ZenHub Board](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/board)

---
## Tasks
- [X] Create [Collaboration Cycle ZenHub Board](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/board)
- [x] Modify Pipelines to reflect [Prototype in Mural](https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1641826404491/e38e110a4b8f1f207926615c80d4f604fbb18128?sender=u5d556df3430960096c525037)
- [ ] Move existing tickets to correct Pipeline
- [ ] Request Peer Review

---

## Acceptance Criteria
- [ ] Collaboration Cycle Zenhub board has been created for usability testing
- [ ] ZenHub board has Pipelines required for usability testing
- [ ] Updates have been Peer Reviewed by Governance Team member
- [ ] User Story need has been fulfilled

---

## Governance Team Resources
[Documentation Update Guidance](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2047868956/WIP+Documentation+Update+Guidance) <br />
[Guidelines for Governance Team Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1127219219/Guidelines+for+Team+spaces) <br />
[Guidelines for Platform Products and Services Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1321140361/Guidelines+for+Platform+Products+and+Services+space) <br />
[Description of Platform Website spaces and content](https://vfs.atlassian.net/wiki/spaces/AP/pages/1241579796/Organization) <br />

---
## How to configure this issue
- [X] **Ensure that story is attached to appropriate Iniative (if not tech debt)** (what body of work is this a part of?)
- [ ] Ensure that any additional labels are added ('tech-debt', 'collaboration-cycle')
- [ ] Ensure that story is in the correct pipeline (Backlog, Planned, In Progress, Review/QA, Closed)
- [ ] Ensure that assignees are correct so you receive all updates/comments on the ticket

**Comment by shiragoodman** [comment]
@chaseakirby in order for Brian and Sean to move forward with usability testing of the core GH ticket, we're going to need to have a Collab Cycle Zenhub board set up. Can you please update the body of this ticket ? and then create the board in Zenhub? thanks!

**Comment by shiragoodman** [comment]
@chaseakirby sounds good - we can discuss tomorrow morning at our 1:1

**Comment by chaseakirby** [comment]
I have created a test [GitHub Projects Beta](https://github.com/orgs/department-of-veterans-affairs/projects/666/views/2?layout=board&visibleFields=%5B%22Title%22%2C%22Assignees%22%2C%22Status%22%2C%22Labels%22%5D) board as a test. So far it seems similar but it there is not support for automatically pulling in issues based on labels. 
It appears to be on GitHub's Future Roadmap, [link #1](https://github.com/github/roadmap/issues/286), [link #2](https://github.com/github/feedback/discussions/5592), and [link #3.](https://github.com/github/feedback/discussions/5528).
In short, the current process of importing issues is an "Add Item" button at the bottom of a column and then searching via issue with "#va.gov-team #ticketnumber (or text)". I'll be continuing to experiment with features and other items worth noting.

**Comment by shiragoodman** [comment]
well that's disappointing... there's not way to have the board filter to issues with a specific label?

**Comment by chaseakirby** [comment]
I haven't found a way yet but I have some more things that I want to try, I'm hoping that it's possible if I can move it into the va.gov-team repo and that may expose the labels that are available somehow.

**Comment by chaseakirby** [comment]
Moving ticket to done, will make updates as a result of testing in later sprints
